services/notify/src/denier.js

```diff
@@ -12,7 +12,9 @@ class Denier {
     }
 
     const address = {notificationType, notificationAddress};
-    return !!await this.DenylistedNotification.load(address, true);
+    let p = await this.db.fns.exists_denylist_address(address);
```

Comment:
If this `p` is for promise, after the `await` it's no longer a promise but an array of rows.  No need to change anything, just a note.

Reply:
I've given it a more descriptive name now.